spack-repo/packages/singularity-eos/package.py

```diff
@@ -218,11 +218,23 @@ def cmake_args(self):
 
         if self.spec.satisfies("@1.9.0:"):
             if "none" not in self.spec.variants["plugins"].value:
-                pdirs = [join_path(self.stage.source_path, self.plugins[p]) for p in self.spec.variants["plugins"].value]
+                pdirs = []
+                for p in self.spec.variants["plugins"].value:
+                  spackage, path = self.plugins[p]
+                  if spackage == "self":
+                      pdirs.append(join_path(self.stage.source_path, path))
+                  else:
+                      pdirs.append(join_path(self.spec[spackage].prefix, path))
```

Comment:
Is this to deal with, e.g., the dust plugin in the examples?

Reply:
Yes. They are part of the singularity-eos source tree.